logtail/logtail_test.go

```diff
@@ -363,7 +363,7 @@ func TestEncode(t *testing.T) {
 	for _, tt := range tests {
 		buf := new(simpleMemBuf)
 		lg := &Logger{
-			timeNow:      func() time.Time { return time.Unix(123, 456).UTC() },
+			clock:        tstest.NewClock(tstest.ClockOpts{Start: time.Unix(123, 456).UTC()}),
```

Comment:
In the spirit of not updating tests that don't actually care about controlling time: keep this one, the others can be left as `tstime.StdClock{}`.

Or go ahead unchanged. _Something_ needed to be specified in the other tests so I guess installing a fake clock is as good a plan as any.

Reply:
the clock is used in this test in the encodeText method in logtail